keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -963,50 +994,43 @@ of this subsystem.  To accomplish these things there are two levels of
 defaulting: one concerning behavior, and one concerning explicit API
 objects.
 
-The effective configuration is the union of (a) the actual API objects
-that exist and (b) implicitly generated backstop objects.  The latter
-are not actual API objects, and might not ever exist as identifiable
-objects in the implementation, but are figments of our imagination
-used to describe the behavior of this subsystem.  These backstop
-objects are implicitly present and affecting behavior when needed.
-There are two implicitly generated RequestPriority backstop objects.
-One is equivalent to the `system-top` object exhibited above, and it
-exists while there is no actual RequestPriority object with `exempt ==
-true`.  The other is equivalent to the `workload-low` object exhibited
-above, and exists while there is no RequestPriority object with
-non-exempt priority.  There are also two implicitly generated
-FlowSchema backup objects.  Whenever a request whose groups include
-`system:masters` is not matched by any actual FlowSchema object, a
-backstop equivalent to the `system-top` object exhibited above is
-considered to exist.  Whenever a request whose groups do not include
-`system:masters` is not matched by any actual FlowSchema object, the
-following backstop object is considered to exist.
+The effective configuration is the union of (a) preserving configuration 
+objects and (b) all the exhibited default configurations above. The 
+request-management system uses the preserving configuration as its initial 
+state on launching when not all the default configuration present in the 
+apiserver's storage and won't reload until ensuring/observing the preserving 
+objects exists. The default configurations will always be created everytime 
+an kube-apiserver restarts.
+
+Additionally, if an administrator disagrees with the default settings and
+feel like replacing with a complete new set. To avoid re-creation of the  
+system defaults above, the administrator is expected to update the objects
+to mute those configurations doesn't meet expectation.
+Administrators who wish to operate with a different configuration can create, 
+update, and delete additional configuration objects and edit the predefined 
+ones; the predefined ones can not be deleted but they can be modified to 
+minimize their effects. In particular, to mute a FlowSchema, you can modify 
+to match no requests. To mute a PriorityLevelConfiguration modify 
+AssuredConcurrencyShares=1. For example:
```

Comment:
Here is a reworded replacement for the paragraph that starts with the incomplete sentence, not referring to "the filter".

>  Administrators can choose to operate with a partially or completely different set of persistent configuration API objects than the default one; note that this does not affect the behavior of the apiserver before it has had a chance to read the persisted configuration API objects. Additional objects can be created, updated, and deleted. The predefined objects can be updated. The predefined objects can not be deleted; administrators can get almost the same effects by updating the predefined objects to minimize their effects. A PriorityLevelConfiguration can be set to have just 1 assured concurrency share; care should be taken to do this only after its queues have drained, otherwise this setting may greatly slow down the rate at which the queues drain. A FlowSchema can be updated to match no requests; following is an example.